src/Profile/Ownerprofile.jsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable no-undef */
+import { useState, useEffect } from "react"
+
+function OwnerProfile() {
+    const [profileDetails, setProfileDetails] = useState({})
+   
+
+
+
+const getOwnerProfileContact = () => {
+    fetch('https://boolean-api-server.fly.dev/Usamaibrahim33%20/contact/1')
+      .then((response) => response.json())
+      .then((data) =>  {
+        console.log('contact data', data)
+        setProfileDetails(data)
+    })
+}
+
+
+    const UpdateDetails = () => {
+        const options = {
+            method: 'PUT',
+            headers: { 'Content-Type' : 'application/json'},
+            body: JSON.stringify(profileDetails)
+        }
+
+
+        fetch('https://boolean-api-server.fly.dev/Usamaibrahim33%20/contact/1', options)
+          .then((response) => response.json())
+          .then((data) => console.log('data posted',data))
+          .catch((error) => console.log(error))
+    }
+
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        UpdateDetails()
+    }
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+        setProfileDetails((prevDetails) => ({
+          ...prevDetails,
+          [name]: value,
+        }));
+      };
+
+
+useEffect(() => {
+    getOwnerProfileContact()
+}, [])
+
+
+
+    return (
+        <>
+        <div className="ownersProfile">
+            <div className="profilehead">
+            <h1> Profile </h1>
+            </div>
+
+            <form className="ownerProfileform" onSubmit={handleSubmit}> 
+       
+            <div className='EachMain-content profilename'>
+                    <div>
+                        <p className=' commentcircle'>us</p>
+                    </div>
+
+                    <div className="profileownername">
+                       <h3>USAMMA IBRAHIM</h3>
+                    </div>
+                   
+                </div>  
+
+
+              
+
+                <div className="formaccount">
+                   <h1>
+                    Account
+                    </h1> 
+
+                    <label htmlFor="firstName">
+                        First Name* <br />
```

Comment:
the form rendering of the fields could be its separate component to make this shorter